src/features/events/hooks/useEventMutations.ts

```diff
@@ -83,6 +83,7 @@ export default function useEventMutations(
     apiClient
       .patch<ZetkinEvent>(`/api/orgs/${orgId}/actions/${eventId}`, data)
       .then((event) => {
+        dispatch(eventLoaded(event));
```

Comment:
This action "lies" to the store by saying an event was loaded, when in fact it was just updated.

Reply:
@kaulfield23 I interpret @richardolsson 's comment here to mean that line 86 should be removed or refactored, but it is still here. Is this conscious or just missed?